### DISCUSSION TITLE
It's not possible to select and deactivate more than one plug in in AG Server

### DISCUSSION THREAD

**Discussion by jeffikovski** [apohl79/audiogridder#401] [Help]
https://youtu.be/S_B6zBh9c5Y 
It would be nice to be able to select and deactivate several plug-ins at once in AG Server.
I kind of expected this behaviour but it's not supported. 
Or is it a bug?

Cheers

**Comment by apohl79** [comment]
I never implemented this, so it is not supported right now.
